### PR TITLE
fix #1075 and replace #1076

### LINKED ISSUE #1075: 使用EngineGroup实现一主多从集群，无法从Master执行Sql查询，并且总是从Slave查询。
使用EngineGroup实现一主多从集群，无法从Master执行Sql查询，并且总是从Slave查询。问题源头已找到：

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

**Comment by lunny** [comment]
will fix #1075 and replace #1076

File: engine_group.go [hunk 1]
```diff
@@ -74,6 +74,13 @@ func (eg *EngineGroup) Close() error {
 	return nil
 }
 
+// NewSession returned a group session
+func (eg *EngineGroup) NewSession() *Session {
```
Comment (BetaCat0) [inline engine_group.go]: I think a name like `NewGroupedSession` or `NewSessionGroup`  will be more clearer to users(instead of comments)

Comment (lunny) [reply engine_group.go]: No.  This `NewSession` will override the `Engine`'s `NewSession`. So people could use `EngineGroup` like `Engine` and don't change anything.

Comment (BetaCat0) [reply engine_group.go]: Got it

**Pull request merged**